### PR TITLE
Add CME Ether futures to symbols database

### LINKED ISSUE #6850: Support All CME Futures Contracts for Crypto
#### Expected Behavior
Supports all CME Crypto Futures

#### Actual Behavior
Only support BTC futures:
> cme,BTC,future,Bitcoin Futures,USD,5.0,5.0,1.0

#### Potential Solution
- Update Market Hours Data Base and Symbol Property Data Base.
    - Add Micro Bitcoin (MBT), Ether (ETH)
    - Revisit micro ether, what's MRB?

#### Reproducing the Problem
Not listed.

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
- [x] I have provided detailed steps to reproduce the issue

### PR DESCRIPTION
**Pull request by jhonabreul**
#### Description

Add[ CME Ether futures](https://www.cmegroup.com/markets/cryptocurrencies/ether/ether.contractSpecs.html) to symbol properties and market hours databases.

Other CME Cryptocurrencies Futurues are already there:
- Bitcoin Futures (BTC)
- Micro Bitcoin Futures (MBT)
- BTIC BTIC on Micro Bitcoin Futures (MIB)
- Micro Ether Futures (MET)
- BTIC BTIC on Micro Ether Futures (MRB)

#### Related Issue

Closes #6850 

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Data/symbol-properties/symbol-properties-database.csv [hunk 1]
```diff
@@ -142,6 +142,7 @@ cme,EI,future,E-mini FTSE Emerging Index Futures,USD,100.0,0.1,1.0
 cme,EMD,future,E-mini S&P MidCap 400 Futures,USD,100.0,0.1,1.0
 cme,ES,future,E-mini S&P 500 Futures,USD,50.0,0.25,1.0
 cme,ESK,future,Euro/Swedish Krona Futures,SEK,125000.0,0.0005,1.0
+cme,ETH,future,Ether Futures,USD,50.0,5.50,1.0
```
Comment (Martin-Molinero) [inline Data/symbol-properties/symbol-properties-database.csv:145]: > ,5.50,

Minimum price variation is probably 0.50?

File: Data/market-hours/market-hours-database.json [hunk 2]
```diff
@@ -22758,6 +22758,260 @@
         "11/25/2022": "12:15:00"
       }
     },
+    "Future-cme-ETH": {
+      "dataTimeZone": "UTC",
+      "exchangeTimeZone": "US/Central",
```
Comment (Martin-Molinero) [inline Data/market-hours/market-hours-database.json:22763]: Just checking, but BTC is `America/Chicago`, micro eth & btc are `America/New_York` & ETH is `US/Central` (we have no other US/Central)?

**Review by Martin-Molinero — CHANGES_REQUESTED** [review]
Couple of comments also, we need to update `FuturesExpiryFunctions` too adding this new future contract 👌

File: Data/market-hours/market-hours-database.json (hunk 2 above)
Comment (jhonabreul) [reply Data/market-hours/market-hours-database.json:22763]: Right, updating with sources:
BTC: https://www.cmegroup.com/markets/cryptocurrencies/bitcoin/bitcoin.contractSpecs.html
Micro BTC (MBT): https://www.cmegroup.com/markets/cryptocurrencies/bitcoin/micro-bitcoin.contractSpecs.html
ETH: https://www.cmegroup.com/markets/cryptocurrencies/ether/ether.contractSpecs.html
Micro ETH (MET): https://www.cmegroup.com/markets/cryptocurrencies/ether/micro-ether.contractSpecs.html

**Review by Martin-Molinero — APPROVED** [review]
Thank you ✨

**Pull request merged**